package.json

```diff
@@ -37,11 +37,12 @@
     "babel-core": "6 || 7 || ^7.0.0-alpha || ^7.0.0-beta || ^7.0.0-rc"
   },
   "devDependencies": {
-    "babel-core": "7.0.0-alpha.17",
-    "babel-plugin-external-helpers": "7.0.0-alpha.17",
-    "babel-plugin-transform-decorators": "7.0.0-alpha.17",
-    "babel-plugin-transform-runtime": "7.0.0-alpha.17",
-    "babel-preset-es2015": "7.0.0-alpha.17",
+    "babel-core": "7.0.0-beta.3",
+    "babel-helper-annotate-as-pure": "7.0.0-beta.3",
```

Comment:
This is not actual dependency of this package, can be removed when new version of babel gets released. This https://github.com/babel/babel/pull/6490 screwed up builds on npm@2